### ISSUE TITLE
O-bike bike list API returns 

### ISSUE THREAD

**Issue by pnteresa** [ubahnverleih/WoBike#14]
GET `https://mobile.o.bike/api/v1/bike/list?latitude=<LATITUDE>&longitude=<LONGITUDE>` returns error message for me:

```
{
    "data": {
        "code": 500,
        "error": "Server is busy,please try later"
    },
    "success": false,
    "errorCode": 400
}
```

Any idea? Thanks.

**Comment by PierrickP** [comment]
Closed, duplicate of #8

**Comment by ubahnverleih** [comment]
short answer: They changed there API and we try to reverse engineering the new API. But its not so easy. You can look up the details at #8 and maybe help
